ES/ESdd009.xml

```diff
@@ -29,134 +28,236 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <msIdentifier>
                   <repository ref="INS0105DD"/>
                   <collection>Ethio-SPaRe</collection>
-                  <idno>DD-009</idno>
+                  <idno facs="ESdd009/009/ESdd009_" n="125">DD-009</idno>
```

Comment:
please check how it has been done e.g. for DD-001 and do in exactly the same way, thank you!

Reply:
Done.